### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by dkey18**
Implemented pagination with search and category filter functions

### CODE REVIEW

**Review by xchanti124 — CHANGES_REQUESTED** [review]
Please apply prettier to all of the files.

If you're using WebStorm, install the Prettier plugin and there should be an option to apply the styling automatically on save.

**Comment by dkey18** [comment]
> Please apply prettier to all of the files.
> 
> If you're using WebStorm, install the Prettier plugin and there should be an option to apply the styling automatically on save.

Should be formated now

File: src/controllers/pageController.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+const Location = require("../models/locationModel");
+
+// @desc Get pages
+// @route GET /api/page
+// FOR PAGE /api/page?pageNumber=1
+// PAGE with SEARCH and CATEGORY example /api/page?pageNumber=1&search=shopping&category=Attractions
+// can be varied as needed
+const pages = asyncHandler(async (req, res) => {
+  const locationsPerPage = 10;
+  let pageNumber = req.query.pageNumber - 1 || 0;
```
Comment (xchanti124) [inline src/controllers/pageController.js]: What's the purpose of `|| 0` here?

**Review by xchanti124 — APPROVED** [review]

File: src/controllers/pageController.js (hunk 1 above)
Comment (dkey18) [reply src/controllers/pageController.js]: If I won’t get a page number from the front end I will use page 0 (1) as standard. I think that code will do this or not?

**Pull request merged**